msal/src/Microsoft.Identity.Client/Internal/Requests/NonInteractive/IWARequest.cs

```diff
@@ -94,8 +94,13 @@ private async Task FetchAssertionFromWsTrustAsync()
                 }
                 else
                 {
-                    throw new MsalException(MsalError.UnknownUserType,
-                        string.Format(CultureInfo.CurrentCulture, MsalErrorMessage.UnsupportedUserType, userRealmResponse.AccountType));
+                    throw new MsalException(
+                        MsalError.UnknownUserType,
+                        string.Format(
+                            CultureInfo.CurrentCulture, 
+                            MsalErrorMessage.UnsupportedUserType, 
+                            userRealmResponse.AccountType, 
+                            this.iwaInput.UserName));
```

Comment:
Is this considered Pii?

Reply:
yes

---
In reply to: [219635839](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1261#discussion_r219635839) [](ancestors = 219635839)

Reply:
ok. we're not logging it.